scripts/qa_crowbarsetup.sh

```diff
@@ -1332,6 +1332,8 @@ function set_node_raid
     disks_count="$3"
 
     wait_for 10 5 "getent hosts $node &> /dev/null" "$node name to be resolvable"
+
+    wait_for 300 10 "timeout -k 20 10 ssh -o UserKnownHostsFile=/dev/null $node true" "$node to become sshable"
```

Comment:
we can probably also do that, but its just testing whether sshd is up, it is not testing whether login succeeds (e.g. ssh auth not yet set up or the like)